### ISSUE TITLE
Update front-end to handle boolean scalars

### ISSUE THREAD

**Issue by robhudson** [mozilla/glam#376]
Boolean scalars have a unique JSON object structure in the `data` attribute. This bug is to update the front-end code to handle the data for booleans.

If you want any JSON object structure changes just let me know.

Commit https://github.com/mozilla/glam/commit/86bbab50 loosened the restriction for booleans so they do not return 404s when we actually have data.

Example JSON structure is below:
```
    [
        {
            "metadata": {
                "channel": "nightly",
                "version": "74",
                "os": "Windows",
                "build_id": "20200110095023",
                "process": "parent",
                "metric": "a11y_backplate",
                "metric_type": "boolean"
            },
            "data": [
                {
                    "percentiles": {
                        "5": 1,
                        "25": 1,
                        "50": 2,
                        "75": 3,
                        "95": 7
                    },
                    "total_addressable_market": null,
                    "client_agg_type": "true"
                },
                {
                    "total_users": 18822,
                    "histogram": {
                        "always": 18821,
                        "never": 1,
                        "sometimes": 0
                    },
                    "total_addressable_market": null
                },
                {
                    "percentiles": {
                        "5": 0,
                        "25": 0,
                        "50": 0,
                        "75": 0,
                        "95": 0
                    },
                    "total_addressable_market": null,
                    "client_agg_type": "false"
                }
            ]
        },
        ....
    ]
```

**Comment by hamilton** [comment]
Thanks @robhudson. Wondering if we should just throw out the `client_agg_type` true / false for now and lean on the histogram data itself. This could change down the line but for now I think it makes the API response a little bit easier.

If we do so, the payload with `histogram` can have the same client_agg_type as a typical categorical histogram (I htink it's `summed-histogram` right?)

**Comment by robhudson** [comment]
So, if the metric type is boolean, don't include the percentiles at all. I imagine the front-end would need to know about this to avoid trying to access percentiles.

I'd shy away from bolting on the client agg type here since the metadata contains everything the front-end would need to conditionally handle boolean types.

**Comment by robhudson** [comment]
Updated in this branch/PR. The charts do not display, however. There's no console error but the spinner never stops spinning.

https://github.com/mozilla/glam/pull/384

**Comment by hamilton** [comment]
Yes, the charts will not display until we tell GLAM about what to do with boolean scalars. This said, we should catch this issue – unknown probe type – and throw an error much earlier on.

#384 should be updated now to render boolean scalars.

**Comment by hamilton** [comment]
Fixed with #384
